src/components/network/network-table.js

```diff
@@ -70,13 +106,24 @@ const NetworkTable = (props) => {
     const [tabIndex, setTabIndex] = React.useState(0);
     const [lineEdit, setLineEdit] = React.useState({});
     const [rowFilter, setRowFilter] = React.useState(undefined);
+    const [popupSelectListName, setPopupSelectListName] = React.useState(false);
+
+    const [listColumnsNames, setListColumnsNames] = useState([]);
+    const [checked, setChecked] = React.useState([]);
+    const [selectedListName, setSelectedListName] = useState([]);
+
```

Comment:
Too many states for the same informations ?

Reply:
checked and selectedListName -> selectedColumnsNames ?

Reply:
No, the checked state is used to memorize the selection/deselection of columns in the dialog window
And, only after clicking on 'save' button, then the selectedListName state is updated